applications/openshift/etcd/etcd_auto_tls/ansible/ansible.yml

```diff
@@ -0,0 +1,15 @@
+# platform = multi_platform_rhel,multi_platform_fedora
```

Comment:
@redhatrises what should we call the OCP ansible files, since there is no shared? (rhel has rule/ansible/shared.yml)

Reply:
@shawndwells call them `shared.yml`